release_notes_filter.pl

```diff
@@ -34,6 +34,7 @@
   'Philippe Marschall' => 'https://github.com/marschall',
   'Rikard Pavelic' => 'https://github.com/zapov',
   'Robert Zenz' => 'https://github.com/RobertZenz',
+  'Robert \'Bobby\' Zenz' => 'https://github.com/RobertZenz',
```

Comment:
Well, Robert did use a different name in the commit author field. I think there's no issue with using this name in the website, so I just use it.

See
```
git show 5b9edb7dfb1b281bffedf7c9f2583f2df354c0

commit 5b9edb7dfb1b281bffedf7c9f2583f2df354c0ea
Author: Robert 'Bobby' Zenz <Robert.Zenz@bonsaimind.org>
Date:   Fri May 19 17:12:51 2017 +0200

    fix: Add fallback to setObject(int, Object) for Number (#812)
...
```